pennylane/optimize/shot_adaptive.py

```diff
@@ -479,7 +478,8 @@ def step(self, objective_fn, *args, **kwargs):
 
             argmax_gamma = np.unravel_index(np.argmax(gamma), gamma.shape)
             smax = max(s[argmax_gamma], 2)
-            self.s[idx] = np.squeeze(np.int64(np.clip(s, min(2, self.min_shots), smax)))
+
+            self.s[idx] = np.int64(np.clip(s, min(2, self.min_shots), smax))

```

Comment:
Somehow the test doesn't catch this change. When I reset this line to it's previous version the test below still passes, both with `qnode` and `expval_cost`.